include/trie.h

```diff
@@ -85,8 +88,20 @@ int trie_add_node(trie_t *t, char current);
 */
 int trie_insert_string(trie_t *t, char *word);
 
+/*
+    Checks if a char exists in a trie 
+    Parameters:
+     - t: A pointer to the given trie
+     - c: The character we want to check
+
+    Returns:
+     - 0 if c exists in t
+     - 1 if it doesn't
```

Comment:
Our other functions aren't really consistent since trie_insert_string returns 0 for success as well. Additionally, the commonly used return value EXIT_SUCCESS (included in <stdlib.h>) is equal to 0.

Reply:
@dwahme

Reply:
0 and 1 stand for EXIT_SUCCESS and EXIT_FAILURE respectively. The function actually returns these values, and the description has been updated accordingly.